### PR TITLE
add a type::chore label to automated addon PRs

### PR DESCRIPTION
**Pull request by laverya**
#### What this PR does / why we need it:

Without a type label, PRs are marked as failing CI.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Review by rrpolanco — APPROVED** [review]

**Review by rrpolanco — DISMISSED** [review]

**Review by emosbaugh — COMMENTED** [review]
@AmberAlston i believe keeping add-ons up to date is considered a chore correct?

**Comment by AmberAlston** [comment]
@emosbaugh Correct. Keeping add-ons up-to-date and patched is currently labeled a chore.

**Comment by emosbaugh** [comment]
@laverya can we get this updated with type::chore so we can merge?

File: .github/workflows/update-containerd.yaml [hunk 1]
```diff
@@ -29,9 +29,15 @@ jobs:
...
           containerd
+          type::chore
         draft: false
         base: "main"
-        body: "Automated changes by the [cron-containerd-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-containerd.yaml) GitHub action"
+        body: |
+          Automated changes by the [cron-containerd-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-containerd.yaml) GitHub action
+          
+          ```release-note
+            Added [Containerd](https://kurl.sh/docs/add-ons/containerd) ${{ steps.update.outputs.containerd_version }}
```
Comment (emosbaugh) [inline .github/workflows/update-containerd.yaml:39]: Can you follow the pattern we use in release notes?

```suggestion
            Adds [Containerd add-on](https://kurl.sh/docs/add-ons/containerd) version(s) ${{ steps.update.outputs.containerd_version }}.

**Review by emosbaugh — CHANGES_REQUESTED** [review]

File: .github/workflows/update-containerd.yaml (hunk 1 above)
Comment (emosbaugh) [reply .github/workflows/update-containerd.yaml:39]: For continerd can you change output for containerd_version to:

```
    echo "::set-output name=containerd_version::${VERSIONS[*]}"
```

File: .github/workflows/update-kubernetes.yaml [hunk 2]
```diff
@@ -31,9 +31,15 @@ jobs:
...
           kubernetes
+          type::chore
         draft: false
         base: "main"
-        body: "Automated changes by the [cron-kubernetes-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-kubernetes.yaml) GitHub action"
+        body: |
+          Automated changes by the [cron-kubernetes-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-kubernetes.yaml) GitHub action
+          
+          ```release-note
+            Adds [Kubernetes](https://kurl.sh/docs/add-ons/kubernetes) version ${{ steps.update.outputs.kubernetes_version }}
```
Comment (emosbaugh) [inline .github/workflows/update-kubernetes.yaml:41]:
```suggestion
            Adds [Kubernetes](https://kurl.sh/docs/add-ons/kubernetes) version(s) ${{ steps.update.outputs.kubernetes_version }}
```

https://github.com/replicatedhq/kURL/pull/3303

File: .github/workflows/update-kotsadm.yaml [hunk 3]
```diff
@@ -30,10 +30,15 @@ jobs:
...
+          type::chore
         draft: false
         base: "main"
-        body: "Automated changes by the [auto-kotsadm-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-kotsadm.yaml) GitHub action"
-
+        body: |
+          Automated changes by the [cron-kotsadm-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-kotsadm.yaml) GitHub action
+          
+          ```release-note
+            Adds [kOTS add-on](https://kurl.sh/docs/add-ons/kotsadm) version ${{ steps.update.outputs.kotsadm_version }}
```
Comment (emosbaugh) [inline .github/workflows/update-kotsadm.yaml:40]:
```suggestion
            Adds [KOTS add-on](https://kurl.sh/docs/add-ons/kotsadm) version ${{ steps.update.outputs.kotsadm_version }}
```

File: .github/workflows/update-sonobuoy.yaml [hunk 4]
```diff
@@ -29,10 +29,15 @@ jobs:
...
+          type::chore
         draft: false
         base: "main"
-        body: "Automated changes by the [cron-sonobuoy-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-sonobuoy.yaml) GitHub action"
-
+        body: |
+          Automated changes by the [cron-sonobouy-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-sonobouy.yaml) GitHub action
+          
+          ```release-note
+            Adds [SonoBouy add-on](https://kurl.sh/docs/add-ons/sonobouy) version ${{ steps.update.outputs.sonobouy_version }}
```
Comment (emosbaugh) [inline .github/workflows/update-sonobuoy.yaml:39]:
```suggestion
            Adds [Sonobouy add-on](https://kurl.sh/docs/add-ons/sonobouy) version ${{ steps.update.outputs.sonobouy_version }}
```

File: .github/workflows/update-velero.yaml [hunk 5]
```diff
@@ -29,10 +29,15 @@ jobs:
...
+          type::chore
         draft: false
         base: "main"
-        body: "Automated changes by the [cron-velero-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-velero.yaml) GitHub action"
-
+        body: |
+          Automated changes by the [cron-veler-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-veler.yaml) GitHub action
+          
+          ```release-note
+            Adds [Velero add-on](https://kurl.sh/docs/add-ons/veler) version ${{ steps.update.outputs.veler_version }}
```
Comment (emosbaugh) [inline .github/workflows/update-velero.yaml:39]:
```suggestion
            Adds [Velero add-on](https://kurl.sh/docs/add-ons/velero) version ${{ steps.update.outputs.velero_version }}
```

File: .github/workflows/update-goldpinger.yaml [hunk 6]
```diff
@@ -31,10 +31,15 @@ jobs:
...
+          type::chore
         draft: false
         base: "main"
-        body: "Automated changes by the [cron-goldpinger-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-goldpinger.yaml) GitHub action"
-
+        body: |
+          Automated changes by the [cron-goldpinger-update](https://github.com/replicatedhq/kURL/blob/main/.github/workflows/update-goldpinger.yaml) GitHub action
+          
+          ```release-note
+            Adds [GoldPinger add-on](https://kurl.sh/docs/add-ons/goldpinger) version ${{ steps.update.outputs.goldpinger_version }}
```
Comment (emosbaugh) [inline .github/workflows/update-goldpinger.yaml:41]:
```suggestion
            Adds [Goldpinger add-on](https://kurl.sh/docs/add-ons/goldpinger) version ${{ steps.update.outputs.goldpinger_version }}
```

**Review by emosbaugh — CHANGES_REQUESTED** [review]

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**